packages/@storefront/core/test/unit/services/tracker.ts

```diff
@@ -72,13 +84,24 @@ suite('Tracker Service', ({ expect, spy, stub, itShouldExtendBaseService }) => {
 
     it('should disable warnings', () => {
       const disableWarnings = spy();
+      const getSessionId = spy();
       const visitorId = app.config.visitorId = '1234';
-      service.client = <any>{ disableWarnings, autoSetVisitor: () => null };
+      service.client = <any>{ disableWarnings, getSessionId, autoSetVisitor: () => null };
 
       service.init();
 
       expect(disableWarnings).to.be.called;
     });
+
+    it('should dispatch a setSessionId action', () => {
+      const disableWarnings = spy();
+      const getSessionId = spy();
+      service.client = <any>{ disableWarnings, getSessionId, autoSetVisitor: () => null };
+
+      service.init();
+
+      expect(app.flux.store.dispatch).to.be.calledWith(SET_SESSION_ID);
```

Comment:
Update assertion to reference `dispatch` directly (see note above).

Reply:
Done.